cavityDisplayGUI.py

```diff
@@ -0,0 +1,119 @@
+from dataclasses import dataclass
+
+from PyQt5.QtGui import QColor
+from PyQt5.QtWidgets import QHBoxLayout
+from epics import PV
+from functools import partial
+from pydm import Display
+from pydm.widgets import PyDMEmbeddedDisplay, PyDMRelatedDisplayButton, PyDMTemplateRepeater
+from typing import List
+
+from cavityWidget import CavityWidget
+from lcls_tools.devices.scLinac import LINAC_OBJECTS
+
+STATUS_SUFFIX = "CUDSTATUS"
+SEVERITY_SUFFIX = "CUDSEVR"
+
+GREEN_FILL_COLOR = QColor(9, 141, 0)
+YELLOW_FILL_COLOR = QColor(244, 230, 67)
+RED_FILL_COLOR = QColor(150, 0, 0)
+PURPLE_FILL_COLOR = QColor(131, 61, 235)
+GRAY_FILL_COLOR = QColor(127, 127, 127)
+
+BLACK_TEXT_COLOR = QColor(0, 0, 0)
+DARK_GRAY_COLOR = QColor(40, 40, 40)
+WHITE_TEXT_COLOR = QColor(250, 250, 250)
+
+DARK_PURPLE_COLOR = QColor(106, 102, 212)
+ALTERNATE_YELLOW_COLOR = QColor(223, 149, 0)
+
+
+@dataclass
+class ShapeParameters:
+    fillColor: QColor
+    borderColor: QColor
+    numPoints: int
+    rotation: float
+
+
+SHAPE_PARAMETER_DICT = {0: ShapeParameters(GREEN_FILL_COLOR, GREEN_FILL_COLOR,
+                                           4, 0),
+                        1: ShapeParameters(YELLOW_FILL_COLOR, YELLOW_FILL_COLOR,
+                                           3, 0),
+                        2: ShapeParameters(RED_FILL_COLOR, RED_FILL_COLOR,
+                                           6, 0),
+                        3: ShapeParameters(PURPLE_FILL_COLOR, PURPLE_FILL_COLOR,
+                                           20, 0),
+                        4: ShapeParameters(GRAY_FILL_COLOR, GRAY_FILL_COLOR,
+                                           10, 0)}
```

Comment:
Were these previously in their own frontEndconstants.py file? It's fine if they're here, but did we eventually want to separate them or would we want to leave them alone in here to avoid the extra .py file floating around?

Reply:
I tried to put these in the constants file and got a bunch of import errors /sob

Reply:
It's functional as is right now, so that's okay then lol